internal/build/lifecycle_execution.go

```diff
@@ -706,6 +744,133 @@ func (l *LifecycleExecution) ExtendRun(ctx context.Context, buildCache Cache, ph
 	defer extend.Cleanup()
 	return extend.Run(ctx)
 }
+func (l *LifecycleExecution) ExtendBuildByDaemon(ctx context.Context) error {
+	extendtime := time.Now()
```

Comment:
As we get closer to completion we could remove some of the profiling stuff in favor of descriptive log messages